syntax = "proto3";

package cafm.identity.permission;

message PermissionAppService_permissionByNameRequest {
  string name = 1;
}

message PermissionAppService_permissionByNameResponse {
  string id = 1;
  string name = 2;
}

message PermissionAppService_permissionsRequest {
  int32 resultFrom = 1;
  int32 resultSize = 2;
}

message PermissionAppService_permissionsResponse {
  string response = 1; 
}


service PermissionAppService {
  rpc permissionByName (PermissionAppService_permissionByNameRequest) returns (PermissionAppService_permissionByNameResponse) {}
  rpc permissions (PermissionAppService_permissionsRequest) returns (PermissionAppService_permissionsResponse) {}
}